TCAT/Utilities/Analytics.swift

```diff
@@ -218,3 +208,14 @@ struct WhatsNewCardDismissedPayload: Payload {
 
     let actionDescription: String
 }
+
+/// Log get errors
+struct GetErrorPayload: Payload {
+    static let eventName: String = "Get Error"
+    let deviceInfo = DeviceInfo()
+    
+    let location: String
+    let type: String 
+    let description: String
+    let url: String?
```

Comment:
Let's take our `url` as well. We currently store `requestURL` to be able to write it to log files that only get used if the user submits feedback to the app. Unfortunately this is all like v legacy code and tbh we like never look at it. Idt the issues our users end up running into r complex enough where we can't repro and need specific logs for it. I'll make an issue to scrap all of the JSONFileManager stuff that uses the url but for now, we can just omit this field on the payload object